app/components/icons/mui/Lock.tsx

```diff
@@ -2,6 +2,7 @@ import { SVGProps } from "react";
 
 const Lock = (props: SVGProps<SVGSVGElement>) => (
   <svg
+    id="icon-lock"
```

Comment:
this id makes querying in Cypress easier.

Reply:
https://github.com/digitalservicebund/grundsteuer/pull/837/files#diff-8a03c18e67afa73f1636e84d1c082b942cc71fc59668d42cb6d0c948640be432R33

Reply:
I think when there is no other way to access the element (e.g. through name) that is totally fine 👍 Thanks for adding the comment to give some context